app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,80 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  before_action :authenticate_user!, except: %i[index show]
+
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all
+  end
+
+  # GET /posts/1
+  # GET /posts/1.json
+  def show; end
```

Comment:
You can remove it since it's being use.

Reply:
Hello Torres,
Thanks for the review Torres. Concerning my partner we are working together asynchronously. We did contact student success earlier this month and that was the solution Iwe were given. I have built this project twice. The first was with devise gem. The review I was given was that you are not supposed to use the gem devise. I rebuild from scratch without the devise gem. I am dissappointed at this point now that you are recommending me to use the devise gem again. How I am going to address this. Please provide some insights because the oding instruction doesnt involve using the device gem